x/feegrant/doc.go

```diff
@@ -0,0 +1,28 @@
+/*
+Package feegrant provides functionality for delegating the payment of transaction fees
+from one account (key) to another account (key).
+
+Effectively, this allows for a user to pay fees using the balance of an account
+different from their own. Example use cases would be allowing a key on a device to
+pay for fees using a master wallet, or a third party service allowing users to
+pay for transactions without ever really holding their own tokens. This package
+provides ways for specifying fee allowances such that delegating fees
+to another account can be done with clear and safe restrictions.
+
+A user would delegate fees to a user using MsgDelegateFeeAllowance and revoke
```

Comment:
Seems like "delegate" is used inconsistently here.  It should be a unidirection relationship but it appears to be used bidirecitonally?

A tx signer, say Alice, might "delegate" the responsibility of paying fees to someone else, say Bob.
But I wouldn't say that Bob in this case is delegating anything to Alice.

So I'd call this MsgAddFeeAllowance.

Reply:
I've been advocating for the language of grant and revoke, so how about `MsgGrantFeeAllowance`?

Reply:
Oh, that is the name of the actual struct. Looks like the docs just need to be updated...